packages/adblocker/tools/stress-test-engine-update.ts

```diff
@@ -168,7 +168,7 @@ async function getMeta(url: string): Promise<any> {
  * Return the CDN URL of a specific revision `rev` (a hash) of subscription `list` (e.g. 'easylist').
  */
 function urlOfRevision(list: string, rev: string): string {
-  return `https://cdn.cliqz.com/adblocker/resources/${list}/${rev}/list.txt`;
+  return `https://cdn.ghostery.com/adblocker/resources/${list}/${rev}/list.txt`;
```

Comment:
Is this intended?

Reply:
I think, the URL is valid. So, it should be okay.

Reply:
cliqz cdn is no longer updated